src/main/java/io/github/mzmine/gui/chartbasics/graphicsexport/GraphicsExportParameters.java

```diff
@@ -39,8 +47,9 @@
     Chart, Plot;
   }
 
-  public static final StringParameter path = new StringParameter("Path", "The file path");
-  public static final StringParameter filename = new StringParameter("Filename", "The file name");
+  public static final DirectoryParameter path = new DirectoryParameter("Path", "The file path");
+  public static final StringParameter filename =

```

Comment:
Wouldn't it make more sense to use the FileParameter instead of separate path + filename?

Reply:
Which FileParameter are you referring to? For the FileNameParameter it would always say the file does not exist.

However, I think it might be easier when exporting multiple graphs to keep the directory loaded and just changing the filename as a string, so you don't have to open the dialog all the time.

Reply:
Please use the FileNameParameter. The same is used in other export modules like CSV export etc. If you want to have the option to remember the last used directory, it is better to add this feature to the FileNameParameter so all export modules can take advantage of it.